app/src/main/java/org/coepi/android/ui/alertsdetails/AlertsDetailsViewModel.kt

```diff
@@ -16,15 +18,30 @@ class AlertsDetailsViewModel(
     val report: LiveData<List<AlertDetailsSymptomViewData>> = just(args.report.toViewData())
         .toLiveData()
 
-    // FIXME temporary title to identify the report. No design yet for this screen.
-    val title: LiveData<String> = just("Report: ${args.report.id}")
+    val title: LiveData<String> = just(toMonthAndDay(args.report.contactTime))
         .toLiveData()
 
+    val exposureTime: LiveData<String> = just(toTime(args.report.contactTime))
+        .toLiveData()
+
+    //TODO Show when the symptoms were reported. Maybe the earliestSymptom Date?
+    val reportedTime: LiveData<String> = just(" ").toLiveData()
+
     private fun Alert.toViewData(): List<AlertDetailsSymptomViewData> =
         emptyList()
 //        report.ids.map { it.toViewData() }
 
     private fun SymptomId.toViewData(): AlertDetailsSymptomViewData =
         // TODO map to localized names
         AlertDetailsSymptomViewData(ic_alert, name, this)
+
+    companion object {
```

Comment:
Any particular reason to put these under an object instead of directly in the class? It's not wrong, but we don't use it in other places.